### PR TITLE
feat: parse genesis file without accessing the node

### PR DESCRIPTION
**Pull request by rmontagnin**
## Description
Superseeds #51 

This PR implements a way to parse the genesis file without accessing the node itself. The following changes have been made:
- renamed the `parse` command to `start`
- renamed the `fix` command to `parse`
- added genesis-file parse subcommand: 
   `juno parse genesis-file` 

**Note**. Version `v3.0.0` of Juno will include this changes, which are **breaking** due to the command renames. Extra precaution needs to be used when updating to this version. 

## Checklist
- [ ] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.  
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

**Review by huichiaotsou — APPROVED** [review]
tested Ack

File: .docs/setup.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Usage:
 Available Commands:
   help        Help about any command
   init        Initializes the configuration files
-  parse       Start parsing the blockchain data
+  start       Start parsing the blockchain data
```
Comment (huichiaotsou) [inline .docs/setup.md:38]: oh the available cmd lacks "parse"

File: cmd/init/cmd.go [hunk 2]
```diff
@@ -13,8 +13,8 @@ const (
 	flagReplace = "replace"
 )
 
-// InitCmd returns the command that should be run in order to properly initialize BDJuno
-func InitCmd(cfg *Config) *cobra.Command {
+// NewInitCmd returns the command that should be run in order to properly initialize BDJuno
```
Comment (MonikaCat) [inline cmd/init/cmd.go]:
```suggestion
// NewInitCmd returns the command that should be run in order to properly initialize Juno
```

**Review by MonikaCat — APPROVED** [review]
tested, LGTM

File: .docs/setup.md (hunk 1 above)
Comment (rmontagnin) [inline .docs/setup.md:38]:
```suggestion
  parse       Parse individual data (eg. the genesis file)
  start       Start parsing the blockchain data
```

Comment (rmontagnin) [reply .docs/setup.md:38]: added now

**Pull request merged**